config/manager/manager.yaml

```diff
@@ -27,7 +27,6 @@ spec:
         - name: manager
           securityContext:
             readOnlyRootFilesystem: true
-            runAsNonRoot: true
```

Comment:
I think this is not the right fix. We shouldn't need to run the image as root.

Instead add:

```golang
     runAsUser: 65534
```

Reply:
NP. The default user in the image is root (uid 0), not great choice by itself. You can override that with runAsUser

Reply:
done